kore/src/Kore/Repl/Data.hs

```diff
@@ -243,7 +243,9 @@ helpText :: String
 helpText =
     "Available commands in the Kore REPL: \n\
     \help                                  shows this help message\n\
-    \claim <n>                             shows the nth claim\n\
+    \claim [n]                             shows the nth claim or if \
+                                           \used without args shows the \
```

Comment:
I think the spacing is wrong, the `\` needs to be below `\claim` and spaces added, no?